src/licensemanager2/workload_managers/slurm/cmd_utils.py

```diff
@@ -294,3 +296,61 @@ async def sacctmgr_modify_resource(
         logger.warning(modify_resource_stdout)
         return False
     return True
+
+
+def return_formated_squeue_out() -> str:
+    """Call squeue via Popen and return the output.
+
+    return type: str
+    """
+
+    try:
+        stdout, _ = subprocess.Popen(
+            [SQUEUE_PATH, "--noheader", "--format='%A|%M|%T'"]
+        )
+    except subprocess.CalledProcessError as e:
+        logger.error(e)
+        raise e
+
+    return stdout.decode().strip()
```

Comment:
Fix the subprocess usage and also fixes mypy problems
```suggestion
     """Call squeue via Popen and return the output."""

    result = subprocess.run(
        [SQUEUE_PATH + " --noheader --format='%A|%M|%T'"],
        capture_output=True,
        shell=True,
        encoding="utf-8"
    )
    if result.returncode != 0:
        logger.error(result.stderr)
        raise Exception(result.stderr)

    return result.stdout.strip()
```

Reply:
The  `raise Exception(result.stderr)` should not be necessary just keeped it because was raising a exception. Do you think it makes sense to raise one here?
And the `CalledProcessError` is not the correct one for this case, check [here](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError)